src/main/java/frc/robot/subsystems/swerve/Module.java

```diff
@@ -64,24 +64,27 @@ public void periodic() {
 
   /** Runs the module closed loop with the specified setpoint state. Returns the optimized state. */
   public SwerveModuleState runSetpoint(SwerveModuleState state) {
-    final var optimizedState = SwerveModuleState.optimize(state, getAngle());
-    return runSetpoint(
-        state,
-        (optimizedState.speedMetersPerSecond - lastSetpoint.speedMetersPerSecond)
-            / (Timer.getFPGATimestamp() - lastTime));
+    return runSetpoint(state, 0.0, 0.0);
```

Comment:
We mean to be running this with no force by default? Looks `accel` *had* previously had a value.

Reply:
the old version calculated a guess for our acceleration target by comparing our current and last velocity setpoint. i have a hunch that this was noisier than it was helpful, but the big issue is the amount of force required varies depending on if the module is trying to linearly accelerate the robot or rotate the robot (motion is resisted by the robot's mass vs moment of inertia). this made the previous acceleration setpoint a poor model. this version directly takes force setpoints from choreo to account for this. however we do not currently have math implemented to handle generating those force setpoints outside of choreo. kP *should* be enough to handle this in most cases where precise acceleration is less of an issue, so having that be the default seems sane to me. having a default of 0 also should make it easier to debug force feedforwards since its easy to turn them off (i would be surprised if there isnt some issue when first running this).